### PR TITLE
feat: Home page + initial commit

### PR DESCRIPTION
**Pull request by manish-GH**
Added the home page. All the navigation is handled by the temporary bottom navbar

Preview: https://gallant-raman-fddb6a.netlify.app/

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+.bottom-nav {
+	display: flex;
+	align-items: center;
+	justify-content: space-around;
+	min-width: 100vw;
+	position: fixed;
+	bottom: 0;
+	left: 0;
+	background-color: var(--theme-bg-white);
+}
```
Comment (swstica) [inline styles.css:122]: You can remove it before the merge.

**Review by swstica — APPROVED** [review]
Rest code looks good!!

File: styles.css (hunk 1 above)
Comment (manish-GH) [reply styles.css:122]: Actually, as of now, I need this for navigation. It's to be removed later that is when I add react into the project

File: screens/Quiz/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+				<a href="./" class="btn btn-link">Quiz</a>
+				<a href="../Rules" class="btn btn-link">Rules</a>
+				<a href="../Result" class="btn btn-link">Result</a>
+			</div>
+		</div>
+		<script>
+			const scrollToTop = () => {
+				document.documentElement.scrollTop = 0;
+			};
+		</script>
```
Comment (kaustubh-jsr) [inline screens/Quiz/index.html:42]: You can put it in a seperate JS file.

File: screens/Quiz/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Quiz</title>
+	</head>
+	<body>
+		<div class="body-container">
+			<nav class="nav-bar">
+				<div class="nav-bar-main">
+					<p onclick="scrollToTop()" class="nav-bar-logo">Cyber Quiz</p>
```
Comment (kaustubh-jsr) [inline screens/Quiz/index.html:21]: Try avoiding inline JS like onclick, you can add them in a JS file.

**Review by vanshitamoolchandani — APPROVED** [review]
Looks good to me. You can merge

**Review by kaustubh-jsr — COMMENTED** [review]
If you want you can refactor the JS snippet. Rest looks good to merge.

File: screens/Quiz/index.html (hunk 3 above)
Comment (manish-GH) [reply screens/Quiz/index.html:21]: I just needed a couple of lines of JS, so it didn't make sense to add a new file for every component also, when we shift to react, I'll anyway have to remove it. So I didn't add it

**Pull request merged**